spaceship/lib/spaceship/test_flight/client.rb

```diff
@@ -75,53 +157,25 @@ def delete_tester_from_group(group_id: nil, tester_id: nil, app_id: nil)
       handle_response(response)
     end
 
-    def get_build(app_id: nil, build_id: nil)
-      assert_required_params(__method__, binding)
-      response = request(:get, "providers/#{team_id}/apps/#{app_id}/builds/#{build_id}")
-      handle_response(response)
-    end
-
-    def put_build(app_id: nil, build_id: nil, build: nil)
-      assert_required_params(__method__, binding)
-      response = request(:put) do |req|
-        req.url "providers/#{team_id}/apps/#{app_id}/builds/#{build_id}"
-        req.body = build.to_json
-        req.headers['Content-Type'] = 'application/json'
-      end
-      handle_response(response)
-    end
-
-    def post_for_testflight_review(app_id: nil, build_id: nil, build: nil)
-      assert_required_params(__method__, binding)
-      response = request(:post) do |req|
-        req.url "providers/#{team_id}/apps/#{app_id}/builds/#{build_id}/review"
-        req.body = build.to_json
-        req.headers['Content-Type'] = 'application/json'
-      end
-      handle_response(response)
-    end
+    ##
+    # @!group Testinfo
+    ##
 
-    def get_groups(app_id: nil)
+    def put_testinfo(app_id: nil, testinfo: nil)
```

Comment:
Do we call this anywhere that it needs to get updated? 

Also, just curious why this is changed to know about the model object rather than passing in the fields?